plugins/org.eclipse.n4js.smith/src/org/eclipse/n4js/smith/TimedDataCollector.java

```diff
@@ -64,7 +62,7 @@ public ClosableMeasurement getClosableMeasurement(String name) {
 	/** This method must be synchronized to protect race conditions when calling data.add */
 	synchronized private void consume(TimedMeasurement measurement) {
 		TimedMeasurement timed = measurement;
-		data.add(new DataPoint(timed.name, timed.elapsed(NANOSECONDS)));
+		// data.add(new DataPoint(timed.name, timed.elapsed(NANOSECONDS)));
```

Comment:
why is this needed? should not affect anything if you run without performance data collection and is undesired if you run with it turned on. So why?

Reply:
Because Luca had turned on performance data collection on Jenkins, by default, leading this line to collecting >700MB `DataPoint`s and crashing the JVM during integration tests.

Reply:
Then this is wrong fix.
Performance data collection should not be turned on by default, exactly because of this reason.